lib/hyacinth/adapters/preservation_adapter/fedora3/rels_int_properties.rb

```diff
@@ -29,8 +29,8 @@ def to(fedora_obj)
 
         def delta_for(resource, fedora_obj, dsid)
           proposed_values = {
-            URIS::EXTENT => [resource.file_size],
-            URIS::HAS_MESSAGE_DIGEST => [resource.checksum ? "urn:#{resource.checksum}" : nil]
+            URIS::EXTENT => [resource && resource.file_size.present? ? resource.file_size : nil],
+            URIS::HAS_MESSAGE_DIGEST => [resource && resource.checksum.present? ? resource.checksum : nil]
```

Comment:
is the urn currently used in existing content?

Reply:
The checksum is stored in Hyacinth resources as a urn, but I've been going back and forth about whether that's good/useful.  What would you prefer?
1) Hyacinth stores "sha256:abcdefg..." and we export to Fedora as "urn:" + hyacinth_checksum
or what I have above:
2) Hyacinth stores "urn:sha256:abcdefg..." and we export to Fedora as hyacinth_checksum
?

Reply:
No, you understand me (I think @cgalarza also asked this elsewhere). I guess one thing to think about would be existing Fedora resources - how they would be imported, and whether importing them would make them appear to be updated even though the checksum doesn't really change.

Reply:
Hmm, that's a good thought.  I'll ticket what you mentioned so we don't forget about it when we implement the existing object import.  I'll mark this thread as resolved after ticket creation, but I re-asked the option 1 vs 2 in another thread with @cgalarza and we can talk more about it there if either of you have any preferences about Hyacinth uri storage format.

Reply:
HYACINTH-556